### PR TITLE
Added test on search functionality

### PR DESCRIPTION
**Pull request by CristianUlmeanu**

### CODE REVIEW

File: src/main/java/autotest/tests/ArticolTest.java [hunk 1]
```diff
@@ -146,6 +146,15 @@ public void verificareTabel50(String titlu, String domeniu, String continut) thr
                 "Articolul care doar ce a fost creat apare in lista cu cele mai recente articole.",
                 "Articolul care doar ce a fost creat nu apare in lista cu cele mai recente articole",
                 true);
+        getDriver().findElement(homeUi.searchHome).sendKeys(titlu);
```
Comment (tr0yyy) [inline src/main/java/autotest/tests/ArticolTest.java:149]: move this in another test method

**Review by tr0yyy — CHANGES_REQUESTED** [review]

File: src/main/java/autotest/tests/ArticolTest.java (hunk 1 above)
Comment (CristianUlmeanu) [reply src/main/java/autotest/tests/ArticolTest.java:161]: Changes implemented.

**Comment by CristianUlmeanu** [comment]
I also created the test for the sorting feature, please check that as well.
Ty :)

File: src/main/java/autotest/tests/ArticolTest.java [hunk 2]
```diff
@@ -146,6 +146,27 @@ public void verificareTabel50(String titlu, String domeniu, String continut) thr
...
+    }
+
+    public void searchArticol(String titlu, String domeniu, String continut) throws IOException, InterruptedException{
+        results.info("Started testing search.",true);
+        Thread.sleep(Duration.of(5, ChronoUnit.SECONDS));
+        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        String currentDate = dtf.format(LocalDateTime.now(ZoneOffset.UTC));
+        HomeUi homeUi = new HomeUi();
+        getDriver().findElement(homeUi.homeNavButton).click();
+        creazaArticol(titlu, domeniu, continut);
```
Comment (tr0yyy) [inline src/main/java/autotest/tests/ArticolTest.java:159]: this functions does not have search logic
a search function should not create a new article everytime

File: src/main/java/autotest/tests/ArticolTest.java [hunk 3]
```diff
@@ -169,28 +190,36 @@ private void modificaArticol(String continut, String continutModificat) throws I
...
+        String continut = "Se testeaza daca elementele de pe pagina principala verifica conditiile.";
         verificareTabel50(titlu, domeniu, continut);
     }
 
+    @Test
+    public void checkSearch() throws InterruptedException,IOException{
+        String titlu = "Test pentru search." + RandomString.make(4);
+        String domeniu = "Istorie";
+        String continut = "Se testeaza searchul.";
+        searchArticol(titlu, domeniu, continut);
```
Comment (tr0yyy) [inline src/main/java/autotest/tests/ArticolTest.java:222]: call createarticol above searcharticol

**Review by tr0yyy — CHANGES_REQUESTED** [review]

**Comment by tr0yyy** [comment]
Rejected due unnecessary tests.

**Pull request closed without merging**